src/test/java/codeu/controller/ChatServletTest.java

```diff
@@ -178,37 +185,72 @@ public void testDoPost_StoresMessage() throws IOException, ServletException {
     Mockito.verify(mockMessageStore).addMessage(messageArgumentCaptor.capture());
     Assert.assertEquals("Test message.", messageArgumentCaptor.getValue().getContent());
 
+      ArgumentCaptor<Activity> activityArgumentCaptor = ArgumentCaptor.forClass(Activity.class);
+      Mockito.verify(mockActivityStore).addActivity(activityArgumentCaptor.capture());
+      Assert.assertEquals("test_username sent a message to test_conversation: Test message.", activityArgumentCaptor.getValue().getDescription());
```

Comment:
`activityArgumentCaptor.getValue()` evaluates to the `Activity` object that was passed to `.addActivity`